app/v2/controllers/requestParsers/validators/ListPaymentsAndAllocationDetailsValidator.scala

```diff
@@ -17,27 +17,36 @@
 package v2.controllers.requestParsers.validators
 
 import api.controllers.requestParsers.validators.Validator
-import api.controllers.requestParsers.validators.validations.{NinoValidation, PaymentLotItemValidation, PaymentLotValidation, StartEndDateValidation}
-import api.models.errors.MtdError
+import api.controllers.requestParsers.validators.validations._
+import api.models.errors.{MtdError, V2_FromDateFormatError, V2_ToDateFormatError}
 import config.AppConfig
 import v2.models.request.listPaymentsAndAllocationDetails.ListPaymentsAndAllocationDetailsRawData
 
 import javax.inject.Inject
 
 class ListPaymentsAndAllocationDetailsValidator @Inject() (appConfig: AppConfig) extends Validator[ListPaymentsAndAllocationDetailsRawData] {
 
-  private val validationSet = List(parameterValidation)
+  private val validationSet = List(parameterValidation, parameterRuleValidation)
 
   private def parameterValidation: ListPaymentsAndAllocationDetailsRawData => List[List[MtdError]] =
     (data: ListPaymentsAndAllocationDetailsRawData) => {
       List(
         NinoValidation.validate(data.nino),
-        StartEndDateValidation.validate(data.dateFrom, data.dateTo),
-        PaymentLotValidation.validate(data.paymentLot),
+        DateFormatValidation.validate(data.fromDate, V2_FromDateFormatError),
+        DateFormatValidation.validate(data.toDate, V2_ToDateFormatError),
+        PaymentLotValidation.validateFormat(data.paymentLot),
         PaymentLotItemValidation.validate(data.paymentLotItem)
```

Comment:
Minor but given the link between payment lot and payment lot item, that one method indicates that it validates format and the other doesn't could be made more obvious. Maybe rename to `PaymentLotItemValidation.validateFormat`. 

Alternatively (more work but arguably nicer) given that `validateMissing` was (arbitrarily?) added to `PaymentLotValidation` rather than `PaymentLotItemValidation` you could ditch `PaymentLotItemValidation` module altogether and move its methods to `PaymentLotValidation`, with `validateMissingPaymentLot`, `validatePaymentLotFormat` and `validatePaymentLotItemFormat` methods.

Reply:
I added the `validateMissing` method to `PaymentLotValidation` because the error code is specifically about the `paymentLot` field, not the `paymentLotItem` field. I think I'll go with the rename of the validate function, and keep the two validations if that's ok.